### PR TITLE
Use sp_runtime::traits::BadOrigin

### PR DESCRIPTION
**Pull request by jasl**
It says `Will be removed after July 2023` but that's not true 😃

### CODE REVIEW

**Comment by jasl** [comment]
@bkchr This one is small 😄

**Review by thiolliere — APPROVED** [review]

**Review by bkchr — APPROVED** [review]
Would be nice if the deprecated error would actually work.

**Review by ggwpez — APPROVED** [review]
>It says `Will be removed after July 2023` but that's not true 😃

It is true, "after" just means any point after that date. I think it is good to keep backwards compatibility for parachain teams as long as possible.

Can you please still add a prdoc that mentions all crates as `patch`? We will add some more convenient way of creating prdocs in the future.

**Comment by jasl** [comment]
> > It says `Will be removed after July 2023` but that's not true 😃
> 
> It is true, "after" just means any point after that date. I think it is good to keep backwards compatibility for parachain teams as long as possible.
> 
> Can you please still add a prdoc that mentions all crates as `patch`? We will add some more convenient way of creating prdocs in the future.

Added

File: prdoc/pr_5011.prdoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Update definition of frozen balance (docs PR)
```
Comment (ggwpez) [inline prdoc/pr_5011.prdoc:4]:
```suggestion
title: "Use `BadOrigin` from `sp_runtime`"
```

**Comment by jasl** [comment]
run cargo fmt manually

File: prdoc/pr_5011.prdoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR refactor usages of deprecated `frame_support::error::BadOrigin` to `sp_runtime::traits::BadOrigin`
+
+crates: 
+- name: pallet-collective-content
+  bump: patch
+- name: polkadot-runtime-common
+  bump: patch
```
Comment (bkchr) [inline prdoc/pr_5011.prdoc:15]:
```suggestion
  bump: patch
- name: polkadot-runtime-parachains
  bump: patch
```

**Review by bkchr — APPROVED** [review]

**Review by acatangiu — APPROVED** [review]

**Pull request merged**